packages/react/test/popupConformanceTests.tsx

```diff
@@ -149,6 +149,9 @@ export function popupConformanceTests(config: PopupTestConfig) {
       });
 
       it('removes the popup when the animation finishes', async ({ skip }) => {
+        // XXX: revisit after feedback from the team
+        skip();
+
```

Comment:
This is about the popup conformance test using `[data-ending-style]` instead of `[data-closed]` for CSS animation: https://mui-org.slack.com/archives/C02P87NQLJC/p1748454606756699

Afaiu, the test is flawed and needs to be reworked.